rel-eng/tito.props

```diff
@@ -403,26 +403,14 @@ whitelist = ansiblerole-insights-client
   rubygem-smart_proxy_remote_execution_ssh
   rubygem-smart_proxy_salt
 
-[katello-nightly-rhel5]
-disttag = .el5
-whitelist = katello-host-tools
-  katello-repos
-  python-rhsm
-  subscription-manager
-
-[katello-nightly-rhel6]
-disttag = .el6
-whitelist = katello-host-tools
-  katello-repos
-
 [katello-nightly-rhel7]
 disttag = .el7
 scl = tfm
 whitelist = katello
   katello-certs-tools
   katello-client-bootstrap
-  katello-host-tools
   katello-installer-base
+  katello-host-tools
```

Comment:
Why are you moving this? Shouldn't we keep it sorted?

Reply:
Accidental move -- its back